pkgs/by-name/ca/cargo-information/package.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, stdenv
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-information";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "hi-rustin";
+    repo = "cargo-information";
+    rev = "v${version}";
+    hash = "sha256-k481iHQ1tVi9fF5/xYR99/1/oRv1nS3WH7W55aPSyfc=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "cargo-test-macro-0.1.0" = "sha256-4u3Ium+WYBdyocuehDulRgUOR74JC6AUI2+A5xlnUGw=";
+    };
+  };
+
+  checkFlags = [
+    # Some tests require network access which is not available in sandboxed Nix builds.
```

Comment:
```suggestion
    # Require network access
```

Reply:
:+1: that's better, https://github.com/NixOS/nixpkgs/compare/b2f3a820395a928da4bb35326a47eb5ecde55483..62be6e81405cfd2dd47caf7074f297995b20618c